### PR TITLE
Add "castle_trouble"

### PR DESCRIPTION
**Pull request by PiMonkeyServers**
Adds the map castle_trouble by Waterbug

### CODE REVIEW

File: castle_trouble/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+team.2.pos = (-11,-5,-43)
+r = 61
+team.1.pos = (11,-5,43)
+initial_stuff =  default:pick_stone, default:torch 3
```
Comment (JostP) [inline castle_trouble/map.conf:4]:
```suggestion
initial_stuff =  default:pick_stone,default:torch 3
```

**Comment by PiMonkeyServers** [comment]
Tested all works as should

**Comment by olliy1or** [comment]
Good map and works as it should

**Review by olliy1or — APPROVED** [review]

**Review by JostP — APPROVED** [review]
Looks good :+1:

**Pull request merged**